### PR TITLE
INJICERT-213- Negative testcases

### PR DESCRIPTION
**Pull request by maheswaras**
Added negative test cases for injicertify APIs

### CODE REVIEW

File: apitest/src/main/resources/injicertify/VCIMockIDANegTC/AuthenticateUserForMockIDANegTC/AuthenticateUserForMockIDANegTC.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      	"transactionId": "$ID:ESignet_OAuthDetailsRequest_ForMockIDA_uin_all_Valid_Smoke_sid_For_Neg_Flow_transactionId$",
+      	"individualId": "6553455",
+      	"authFactorType" : "OTP",
+      	"challenge" : "111111",
+        "format": "alpha-numeric",
+        "sendOtp":{
+    "encodedHash": "$ID:ESignet_OAuthDetailsRequest_ForMockIDA_uin_all_Valid_Smoke_sid_For_Neg_Flow_encodedResp$",
+    "requestTime": "$TIMESTAMP$",
+    "transactionId": "$ID:ESignet_OAuthDetailsRequest_ForMockIDA_uin_all_Valid_Smoke_sid_For_Neg_Flow_transactionId$",
+	"individualId": "6553455",
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/VCIMockIDANegTC/AuthenticateUserForMockIDANegTC/AuthenticateUserForMockIDANegTC.yml:22]: Indentation is not correct

File: apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml [hunk 2]
```diff
@@ -0,0 +1,680 @@
...
+      checkErrorsOnlyInResponse: true
+      restMethod: post
+      validityCheckRequired: true
+      inputTemplate: injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC2
+      outputTemplate: injicertify/error2
+      input: '{
+      	"client_id": "$ID:ESignet_CreateOIDCClientV2_ForMockIDA_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateToken_ForMockIDA_Valid_Smoke_sid_For_Neg_Flow_access_token$",
+        "format": "ldp_vc",
+		"type": [{types: "VerifiableCredential"}, {types: "MockVerifiableCredential"}],
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml:272]: Indentation is not correct

File: apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml [hunk 3]
```diff
@@ -0,0 +1,680 @@
...
+      validityCheckRequired: true
+      inputTemplate: injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC
+      outputTemplate: injicertify/error2
+      input: '{
+      	"client_id": "$ID:ESignet_CreateOIDCClientV2_ForMockIDA_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateToken_ForMockIDA_Valid_Smoke_sid_For_Neg_Flow_access_token$",
+        "format": "ldp_vc",
+      	"type": [{types: "VerifiableCredential"}, {types: "MockVerifiableCredential"}],
+      	"@context": [{context: "$VCICONTEXTURL$"}],
+		"proof_type": "jwt"
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml:320]: Indentation is not correct

File: apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml [hunk 4]
```diff
@@ -0,0 +1,680 @@
...
+      inputTemplate: injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC
+      outputTemplate: injicertify/error2
+      input: '{
+      	"client_id": "$ID:ESignet_CreateOIDCClientV2_ForMockIDA_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateToken_ForMockIDA_Valid_Smoke_sid_For_Neg_Flow_access_token$",
+        "format": "ldp_vc",
+      	"type": [{types: "VerifiableCredential"}, {types: "MockVerifiableCredential"}],
+      	"@context": [{context: "$VCICONTEXTURL$"}],
+		"proof_type": " ",
+		"proof_jwt": "$PROOF_JWT_3$"
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml:368]: Indentation is not correct

File: apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml [hunk 5]
```diff
@@ -0,0 +1,680 @@
...
+      validityCheckRequired: true
+      inputTemplate: injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC
+      outputTemplate: injicertify/error2
+      input: '{
+      	"client_id": "$ID:ESignet_CreateOIDCClientV2_ForMockIDA_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateToken_ForMockIDA_Valid_Smoke_sid_For_Neg_Flow_access_token$",
+        "format": "ldp_vc",
+      	"type": [{types: "VerifiableCredential"}, {types: "MockVerifiableCredential"}],
+      	"@context": [{context: "$VCICONTEXTURL$"}],
+		"proof_jwt": "$PROOF_JWT_3"
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml:390]: Indentation is not correct

File: apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml [hunk 6]
```diff
@@ -0,0 +1,680 @@
...
+      inputTemplate: injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC
+      outputTemplate: injicertify/error2
+      input: '{
+      	"client_id": "$ID:ESignet_CreateOIDCClientV2_ForMockIDA_all_Valid_Smoke_sid_clientId$",
+        "idpAccessToken": "$ID:ESignet_GenerateToken_ForMockIDA_Valid_Smoke_sid_For_Neg_Flow_access_token$",
+        "format": "ldp_vc",
+      	"type": [{types: "VerifiableCredential"}, {types: "MockVerifiableCredential"}],
+      	"@context": [{context: "$VCICONTEXTURL$"}],
+		"proof_type": "jwt",		
+		"proof_jwt": ""
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/VCIMockIDANegTC/GetCredentialForMockIDANegTC/GetCredentialForMockIDANegTC.yml:413]: Indentation is not correct

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

File: apitest/src/main/resources/injicertify/VCIMockIDANegTC/OAuthDetailsRequestForMockIDANegTC/OAuthDetailsRequestForMockIDANegTC.yml [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+      	"scope": "mock_identity_vc_ldp",
+      	"responseType": "code",
+      	"redirectUri": "$IDPREDIRECTURI$",
+      	"display": "popup",
+	    "prompt": "login",
+	    "acrValues": "mosip:idp:acr:generated-code",
+	    "nonce": "973eieljzng",
+	    "state": "eree2311",
+	    "claimsLocales": "en",
+	    "codeChallenge": "$CODECHALLENGE$",
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/VCIMockIDANegTC/OAuthDetailsRequestForMockIDANegTC/OAuthDetailsRequestForMockIDANegTC.yml:21]: Indentation is not correct

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

**Review by nandhu-kumar — APPROVED** [review]

File: apitest/testNgXmlFiles/injicertifyApi.xml [hunk 8]
```diff
@@ -477,5 +477,49 @@
 			<class name="io.mosip.testrig.apirig.testscripts.SimplePost" />
 		</classes>
 	</test>
+	
+		<test name="OAuthDetailsRequestVCIForMockIDANegTC">
```
Comment (mohanachandran-s) [inline apitest/testNgXmlFiles/injicertifyApi.xml:481]: Correct the indentation for this line

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

**Review by jainhitesh9998 — APPROVED** [review]

**Pull request merged**